homeassistant/components/yamaha/media_player.py

```diff
@@ -86,6 +91,8 @@
     {
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
         vol.Optional(CONF_HOST): cv.string,
+        vol.Optional(CONF_VOLUME_DB_MIN, default=DEFAULT_MIN_VOLUME): vol.Coerce(float),
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler